client/state/happychat/middleware.js

```diff
@@ -67,6 +132,14 @@ export default function( connection = null ) {
 
 	return store => next => action => {
 		switch ( action.type ) {
+			case HAPPYCHAT_CONNECTING:
+				if ( getHappychatConnectionStatus( store.getState() ) === 'connected' ) {
+					// If chat is already connected, do nothing and stop the action from proceeding to reducers
+					return;
```

Comment:
Thanks. I probably jumped the gun in merging this PR instead of waiting on feedback on this issue. I'll add this on my to-do list and ping y'all when it's ready for review.